### PR TITLE
Changes required during the netpyne migration to geppetto meta

### PR DESCRIPTION
**Pull request by ddelpiano**

### CODE REVIEW

**Comment by ddelpiano** [comment]
@filippomc for the changes for the widget factory I will do dedicated PR just for that.

File: geppetto.js/geppetto-client/src/components/ComponentFactory.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+const ComponentFactory = {
```
Comment (filippomc) [inline geppetto.js/geppetto-client/src/components/ComponentFactory.js:4]: is everything in the ComponentFactory actually needed? I remember that only a bit of the functionality is really used by the PythonCapability, but I may be wrong

File: geppetto.js/geppetto-client/src/common/layout/WidgetFactory.tsx [hunk 2]
```diff
@@ -56,7 +56,9 @@ class WidgetFactory {
     }
     return Component ?
       <Component
+        id={widgetConfig.id}
         key={widgetConfig.id}
+        model={widgetConfig.model}
```
Comment (filippomc) [inline geppetto.js/geppetto-client/src/common/layout/WidgetFactory.tsx:61]: @ddelpiano did you try to move the model inside config?

Comment (ddelpiano) [reply geppetto.js/geppetto-client/src/common/layout/WidgetFactory.tsx]: https://github.com/MetaCell/geppetto-meta/pull/255/commits/1c0318c7410c2d8faa6dc3d53bb3ea6c76629947

File: geppetto.js/geppetto-client/src/components/ComponentFactory.js (hunk 1 above)
Comment (ddelpiano) [reply geppetto.js/geppetto-client/src/components/ComponentFactory.js:4]: the rest is used by jupyter_geppetto, we actually need everything in there.

**Review by filippomc — APPROVED** [review]
Looks good to me

**Pull request merged**